flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Engine.js

```diff
@@ -64,83 +64,32 @@ var A320_Neo_LowerECAM_Engine;
             this.updateIGN();
         }
         updateIGN() {
-            let ignLeftTargetState = A320_Neo_LowerECAM_Engine.Definitions.IGN_STATE.NONE;
             let engineStarting = (SimVar.GetSimVarValue("GENERAL ENG STARTER:1", "bool") === 1) ? true : false;
             let n2Igniting = (SimVar.GetSimVarValue("TURB ENG IS IGNITING:1", "bool") === 1) ? true : false;
             let n2Percent = SimVar.GetSimVarValue("ENG N2 RPM:1", "percent");
-            if (engineStarting && n2Igniting && n2Percent > 18 && n2Percent < 55) {
-                if (this.ignRightCurrentState == A320_Neo_LowerECAM_Engine.Definitions.IGN_STATE.A) {
-                    ignLeftTargetState = A320_Neo_LowerECAM_Engine.Definitions.IGN_STATE.B;
-                } else {
-                    ignLeftTargetState = A320_Neo_LowerECAM_Engine.Definitions.IGN_STATE.A;
-                }
-            }
+            this.igniterLeft.updateStatus(engineStarting, n2Igniting, n2Percent);
             if (n2Percent > 50) {
                 // Close left valve
                 this.engineLeft.setEngineBleedValveState(false, false);
             }
-            let ignRightTargetState = A320_Neo_LowerECAM_Engine.Definitions.IGN_STATE.NONE;
+
             engineStarting = (SimVar.GetSimVarValue("GENERAL ENG STARTER:2", "bool") === 1) ? true : false;
             n2Igniting = (SimVar.GetSimVarValue("TURB ENG IS IGNITING:2", "bool") === 1) ? true : false;
             n2Percent = SimVar.GetSimVarValue("ENG N2 RPM:2", "percent");
```

Comment:
**issue**: Not necessarily for this PR, but something to note: `ENG N2 RPM` is affected by outside air temperature and altitude (amongst other things). It might be that when you set the sim's environment to the coldest OAT possible, engine idle N2 will never go above `55`.

I ran into this myself and another dev team member pointed out that `TURB ENG CORRECTED N2` is the better sim variable to use, as it is adjusted for outside influences.

I'll leave it up to you to decide if you'd like to test and (maybe) change this within this PR. Or if you'd like to move it to another PR 😀.

Reply:
Sounds like something that would be good to fix, but I do think it is outside of the scope of this PR. Perhaps you could create a bug to track it?

Reply:
Feel free to open a bug if you want. Was more of an FYI.